File: Ejercicio Final/sistema de pagos de servicios/Entel/Factura.js
// Factura.js
const { Sequelize } = require('sequelize');

const sequelize = new Sequelize('entel', 'root', '', {
    host: 'localhost',
    dialect: 'mysql',
    port: 3306,
});

const Factura = sequelize.define('Factura', {
    nroFactura: {
        type: Sequelize.STRING,
        primaryKey: true
    },
    empresa: {
        type: Sequelize.STRING,
        defaultValue: 'Entel'
    },
    ci: {
        type: Sequelize.STRING,
        allowNull: false
    },
    nombreCompleto: {
        type: Sequelize.STRING,
        allowNull: false
    },
    periodo: {
        type: Sequelize.STRING,
        allowNull: false
    },
    monto: {
        type: Sequelize.FLOAT,
        allowNull: false
    },
    estado: {
        type: Sequelize.STRING,
        defaultValue: 'Pendiente'
    }
}, {
    tableName: 'facturas',
    timestamps: false
});

module.exports = { Factura, sequelize };

File: Ejercicio Final/sistema de pagos de servicios/PlataformaPagosSOAP/PlataformaPagosSOAP/Factura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PlataformaPagosSOAP
{
    public class Factura
    {
        public string empresa { get; set; }
        public string nroFactura { get; set; }
        public string ci { get; set; }
        public string nombreCompleto { get; set; }
        public string periodo { get; set; }
        public double monto { get; set; }
        public string estado { get; set; }
    }

}